### ISSUE TITLE
failed to parse the FASTA/FASTQ record 

### ISSUE THREAD

**Issue by AJ211** [marbl/Winnowmap#27]
Dear @cjain7 ,
  When I mapped the ONT reads to GRCh38 without alt sequence using winnowmap with version 2.0.3, it occured the warning message just like "failed to parse the FASTA/FASTQ record next to '783361e5-baa7-44de-9c84-4d1399aa5ceb".  So what does this warning mean? Does it affect the final sam file. My command line shows as below:

  winnowmap -W repetitive_k15.txt  --cs --MD -L  -t 32 -ax map-ont  ${ref}  ${fq}  > out.sam

Thanks in advance
Jiao

**Comment by cjain7** [comment]
Can you also try minimap2 and see if the same warning is shown?

**Comment by cjain7** [comment]
Also, you may want to check the fastq file record to make sure it's properly formatted
One read is probably left unmapped in SAM file as per this warning.

**Comment by AJ211** [comment]
> Can you also try minimap2 and see if the same warning is shown?

Yes, I ran minimap2 with the same fastq file, and it showed the similar error "wrong FASTA/FASTQ record". Maybe some wrong goes to my fastq. I'll check it later. Thank you.
